docs/modules/mobile/reconnection-logic.md

```diff
@@ -0,0 +1,74 @@
+<%(TOC)%>
+
+# Reconnection logic
+
+Mobile applications present new challenges in terms of handling client connections. A client may disconnect due to unstable connections or when pausing the application. We want to dive a bit deeper into the causes of a disconnect and how you can handle it in your game.
+
+## Why clients disconnect
+
+### Unstable connection / cellular reception
```

Comment:
We say cellular reception here but then never reference it in the section below

Reply:
removing it